src/serialization/RDB/decoder/previous/v0/decode_v0.c

```diff
@@ -2,19 +2,12 @@
 #include "assert.h"
 
 void *RAI_RDBLoadTensor_v0(RedisModuleIO *io) {
-    int64_t *shape = NULL;
-    int64_t *strides = NULL;
-
     DLDevice device;
     device.device_type = RedisModule_LoadUnsigned(io);
     device.device_id = RedisModule_LoadUnsigned(io);
-    if (RedisModule_IsIOError(io))
-        goto cleanup;
```

Comment:
Since `size_t ndims = RedisModule_LoadUnsigned(io)` returns 0 in case of io error, I thought that it will be discovered down the road without causing an error, and we will find the error when trying to load the blob.